lib/Doctrine/ODM/PHPCR/Translation/TranslationStrategy/TranslationStrategyInterface.php

```diff
@@ -122,7 +122,7 @@ public function getTranslatedPropertyPath($alias, $propertyName, $locale);
      */
     public function alterQueryForTranslation(
         QueryObjectModelFactoryInterface $qomf,
-        SelectorInterface &$selector,
+        SourceInterface &$selector,
```

Comment:
this is a BC break if somebody has his own translation strategy. can you add a note on this to CHANGELOG.md please? i think we still need to break it - its not very likely people have their own strategy i hope, and we really should fix this.